package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "pp-frontend",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```

Comment:
Do we really want ESlint? It causes me nothing but headaches whenever I encounter it